index.js

```diff
@@ -1,4 +1,91 @@
-//cat's are always hungry
-var now = new Date();
-// the cat says
-console.log('Feed the kitties at ' + now);
+window.oranjs = window.oranjs || {};
+
+oranjs.Globals = {
```

Comment:
I'm familiar with ES6 but not familiar enough with classes so I thought it's better to get in the base functionality first. 😅 
Even though it's just a fun project, expanding the feature set a bit would be really fun IMO :)
Something like "fake pet AI". Sort of Tamagochi on the web